v2/cli/init_command.go

```diff
@@ -95,19 +95,26 @@ func initialize(c *cli.Context) error {
 
 	// If git repo is provided, create a separate js/ts file for script
 	var overwrite bool
+	var fileWrite bool
 	if fields.GitCloneURL != "" {
 		scriptPath := path.Join(".brigade", brigadeScriptFileName)
-		if overwrite, err = checkFileOverwrite(scriptPath); err != nil {
+		if fileWrite, overwrite, err =
+			checkFileOverwrite(scriptPath); err != nil {
 			return err
 		}
-		if overwrite {
+		if fileWrite {
 			if err = ioutil.WriteFile(
 				scriptPath,
 				scriptBytes,
 				0644,
 			); err != nil {
 				return err
 			}
+			if overwrite {
+				fmt.Println("Overwriting " + brigadeScriptFileName + "...")
```

Comment:
Can we use `scriptPath` here instead of `brigadeScriptFileName`? Same comment applies on line 116.

Reply:
Also, let's opt for `fmt.Printf("Overwriting %s...\n", scriptPath)` over using `+` for string concat. Also applies in similar spots.